Test/test/main/CalculatorDateAndTimeTest.java

```diff
@@ -0,0 +1,113 @@
+package main;
+import org.junit.Before;
+import org.junit.Test;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
+
+import static org.junit.Assert.*;
```

Comment:
arent you importing just AssertEquals? if so, then no need to import every method from Assert.